src/middlewares/authMiddleware.js

```diff
@@ -0,0 +1,27 @@
+const { ApiError } = require('../exceptions/ApiError.js');
+const { jwtService } = require('../services/jwt.service.js');
+
+function authMiddleware(req, res, next) {
+  const authHeader = req.headers['authorization'];
+
+  if (!authHeader) {
+    throw ApiError.Unauthorized();
+  }
+
+  const [, accessToken] = authHeader.split(' ');
```

Comment:
Consider adding a check to ensure the authorization header is in the expected 'Bearer <token>' format before splitting it. This will prevent potential errors if the header format is incorrect.